getting_started/first_3d_game/03.player_movement_code.rst

```diff
@@ -369,6 +367,11 @@ Child Scene*.
 In the popup, double-click ``player.tscn``. The character should appear in the
 center of the viewport.
 
+Select the newly instanciated Player node, in the Inspector tab under Node 
+section, load the previously created player script i.e. Player.gd.
```

Comment:
```suggestion
section, load the previously created player script i.e. ``Player.gd``.
```

Reply:
Though this is not correct, the script is already attached, as per:
> Right-click the ``Player`` node and select *Attach Script* to add a new script to
it. In the popup, set the *Template* to *Empty* before pressing the *Create*
button.

So you must have missed this instruction if you feel this is needed to be added here?

Reply:
The player script was already attached in the player scene. But after instancing the player in the main scene and running the main scene, the controls were not working. That's when I saw the script was empty in the Inspector when selecting the player node in the main scene. After attaching it there, I was able to control the character in the main scene.